### PR TITLE
Use nvme for etcd for shiftstack

### PR DESCRIPTION
**Pull request by akrzos**

### CODE REVIEW

**Comment by akrzos** [comment]
Addresses #282

**Comment by masco** [comment]
Can one of the admins verify this patch?

**Review by venkataanil — COMMENTED** [review]
Hi Alex, I have pushed this PR https://github.com/redhat-performance/jetpack/pull/301/ long back for the same. But it is not merged as not getting enough reviews. Kindly review it.

@akrzos @smalleni we are OK to merge any of PRs 301 and 314, but better to avoid duplicate efforts.

**Comment by akrzos** [comment]
> Hi Alex, I have pushed this PR #301 long back for the same. But it is not merged as not getting enough reviews. Kindly review it.
> 
> @akrzos @smalleni we are OK to merge any of PRs 301 and 314, but better to avoid duplicate efforts.

Whichever one you prefer but I think the one liner in #301 will be a bit difficult to maintain or untangle when things go sideways. Just my .02 cents.

File: ocp_on_osp.yml [hunk 1]
```diff
@@ -183,6 +183,43 @@
...
+      with_items:
+        - master
+        - worker
+        - bootstrap
+
+    - name: Read original master.ign config file
+      slurp:
+        src: "/home/stack/{{ ocp_cluster_name }}/master.ign.orig"
+      register: master_ign
+      when: ocp_master_etcd_nvme
```
Comment (venkataanil) [inline ocp_on_osp.yml:206]: where is this ocp_master_etcd_nvme defined? Can you please add it to vars file and add a description for it as a comment?

File: ocp_on_osp.yml [hunk 2]
```diff
@@ -183,6 +183,43 @@
...
+
+    - name: Get master etcd on nvme ignition configuration
+      set_fact:
+        master_ign_config_base: "{{ master_ign['content'] | b64decode }}"
+        master_etcd_nvme_config: "{{ lookup('template', 'master-etcd.ign.json') }}"
```
Comment (venkataanil) [inline ocp_on_osp.yml:211]: Can you please submit master-etcd.ign.json file?

File: ocp_on_osp.yml (hunk 1 above)
Comment (akrzos) [reply ocp_on_osp.yml:206]: The vars are [here](https://github.com/redhat-performance/jetpack/blob/13c1ac023fed5cf871c453789ad515ddf9814773/vars/shift_stack_vars.yaml#L29-L31).  I actually had them in the multiple create/delete shiftstack commit and I think rebasing merged them "out" of this commit if that makes sense.

File: ocp_on_osp.yml (hunk 2 above)
Comment (akrzos) [reply ocp_on_osp.yml:211]: The file is [here](https://github.com/redhat-performance/jetpack/blob/13c1ac023fed5cf871c453789ad515ddf9814773/templates/master-etcd.ign.json).

**Review by venkataanil — COMMENTED** [review]
Thanks Alex.

**Pull request merged**